### PR TITLE
Translate language & plugin settings improvement

### PR DESCRIPTION
**Pull request by YU000jp**
1. [x] Translate language by using Logseq-l10n (https://github.com/sethyuan/logseq-l10n)
   - [x] Chinese (zh-CN)
     > I don't understand Chinese. I had ChatGPT translate it based on the Japanese translation I made earlier. So please do the validation for this without me.
   - [x] Japanese
1. Plugin settings
   - [x] Organizing descriptions

### CODE REVIEW

**Review by sywhb — COMMENTED** [review]
@YU000jp Thank you!

One thing is we are not using `pnpm` right now but we are considering it

**Comment by YU000jp** [comment]
Added
- [ ] Improvement of "README"
- [ ] Japanese translation

**Review by sywhb — APPROVED** [review]
LGTM

File: src/translations/zh-CN.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "Enter the template to use for new articles": "输入要用于新文章的模板",
+    "The template to use for new articles.": "新文章的模板。",
+    "[> Refer to this doc for more info](https://docs.omnivore.app/integrations/logseq.html#controlling-the-layout-of-the-data-imported-to-logseq) [> Variables available could be found here](https://docs.omnivore.app/integrations/logseq.html#variables-available-to-the-highlight-template)": "[> 查看此文档以获取更多信息](https://docs.omnivore.app/integrations/logseq.html#controlling-the-layout-of-the-data-imported-to-logseq) [> 可用变量在此处找到](https://docs.omnivore.app/integrations/logseq.html#variables-available-to-the-highlight-template)",
+    "Enter the template to use for new highlights": "输入要用于新高亮的模板",
+    "The template to use for new highlights.": "新高亮的模板。",
+    "Advanced Settings": "高级设置",
+    "API Endpoint": "API 终点",
+    "Enter the Omnivore server's API endpoint here (default: https://api-prod.omnivore.app/api/graphql )": "在此处输入 Omnivore 服务器的 API 终点（默认: https://api-prod.omnivore.app/api/graphql ）",
+    "Failed to create Omnivore page. Please check the pageName in the settings": "创建 Omnivore 页面失败。请检查设置中的页面名称"
```
Comment (sywhb) [inline src/translations/zh-CN.json:46]: 👍

**Pull request merged**